### DISCUSSION TITLE
What about adding Z tests (mean & proportion) to pingouin?

### DISCUSSION THREAD

**Discussion by aurel-p** [raphaelvallat/pingouin#296] [Ideas]
Hello,

Are you planning to add functions for both mean and proportion Z tests, like in statsmodels but with additional outputs (achieved power, diff CI, effet size, ...)? Although these are basics, it would be convenient to include them into pingouin.

**Comment by raphaelvallat** [comment]
hi,

I wasn't planning on adding these functions in the near future, but please feel free to open an issue with the proposed new features. If you do so, can you please provide an example of these functions in statsmodels or R, as well as how you would envision this function to be implemented in Pingouin (i.e. what would calling the function look like).

Thank you!
Raphael
